### PR TITLE
feat: [TOL-2175] Pass isLocalePublishingEnabled flag via instance param

### PR DESCRIPTION
**Pull request by colomolo**
Pass flag for further use dependant components.

### CODE REVIEW

File: packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx [hunk 1]
```diff
@@ -92,8 +92,12 @@ export function FetchingWrappedAssetCard(props: FetchingWrappedAssetCardProps) {
       renderDragHandle: props.renderDragHandle,
       onEdit,
       onRemove,
-      useLocalizedEntityStatus: props.sdk.parameters.instance.useLocalizedEntityStatus,
-      isLocalized: !!('localized' in props.sdk.field && props.sdk.field.localized), // missing in types :(
+      useLocalizedEntityStatus:
+        props.sdk.parameters.instance.useLocalizedEntityStatus &&
+        props.sdk.parameters.instance.isLocalePublishingEnabled,
```
Comment (chrishelgert) [inline packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx:97]: `useLocalizedEntityStatus` should be the same as `isLocalePublishingEnabled`?

useLocalizedEntityStatus resolves to the feature flag / entitlement check

Comment (colomolo) [reply packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx:97]: Where is that being set?

Comment (chrishelgert) [reply packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx:97]: web app: src/javascripts/app/entity_editor/EntityField/useWidgetApi.ts#L226

**Comment by chrishelgert** [comment]
`props.sdk.parameters.instance.isLocalePublishingEnabled` should the remaining ones be `props.sdk.parameters.instance.useLocalizedEntityStatus` too?

**Pull request closed without merging**